manifests/shared/deployments/ipsec.yml

```diff
@@ -0,0 +1,9 @@
+meta:
+  ipsec:
+    targets:
+    - 10.0.32.0/24
```

Comment:
Actually, it's not 3 it's four:
- https://github.com/alphagov/paas-cf/blob/master/terraform/cloudfoundry/variables.tf#L25
- https://github.com/alphagov/paas-cf/blob/ipsec_release_131802383/terraform/cloudfoundry/outputs.tf#L149
- https://github.com/alphagov/paas-cf/blob/ipsec_release_131802383/manifests/shared/spec/fixtures/terraform/terraform-outputs.yml#L48
- https://github.com/alphagov/paas-cf/blob/ipsec_release_131802383/manifests/shared/deployments/ipsec.yml#L9

Looks ugly.